optaplanner-quarkus-integration/optaplanner-quarkus-benchmark/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,16 @@
+---
+name: "OptaPlanner AI constraint solver"
+metadata:
+  keywords:
+    - "optaplanner"
+    - "kogito"
+    - "constraint-solver"
+    - "artificial-intelligence"
+    - "employee-rostering"
+    - "vehicle-routing-problem"
+    - "maintenance-scheduling"
+    - "timetabling"
+  guide: "https://quarkus.io/guides/optaplanner"
```

Comment:
Another extensions implies a different name and keywords, wouldn't you say?

Reply:
+1

Can the guide be the same? I think it can, but it doesn't have to be.

Reply:
The use case keywords should only be for quarkus-optaplanner itself.